bld/namelist_files/use_cases/ssp585_cam6_noresm_tropstratchem.xml

```diff
@@ -0,0 +1,296 @@
+<?xml version="1.0"?>
+
+<namelist_defaults>
+
+<!--micro_do_sb_physics>   .true.   </micro_do_sb_physics-->
+<!--do_tms>                .false.  </do_tms-->
+<!--do_beljaars>           .true.   </do_beljaars-->
+
+<history_aerosol>.true.</history_aerosol>
+<modal_strat_sulfate>.true.</modal_strat_sulfate>
+<lght_no_prd_factor hgrid="0.9x1.25"> 1.0 </lght_no_prd_factor>
+<lght_no_prd_factor hgrid="1.9x2.5"> 10.0 </lght_no_prd_factor>
+
+<!-- Solar constant -->
+<solar_irrad_data_file>atm/cam/solar/SolarForcingCMIP6_18491230-22991231_c171031.nc</solar_irrad_data_file>
```

Comment:
Should this file path have `$INPUTDATA_ROOT`?
```suggestion
<solar_irrad_data_file>$INPUTDATA_ROOT/atm/cam/solar/SolarForcingCMIP6_18491230-22991231_c171031.nc</solar_irrad_data_file>
```

Reply:
Hi, for the moment I suggest not to introduce $INPUTDATA_ROOT.